core/config/config.go

```diff
@@ -364,10 +364,11 @@ func migrateAccessTokens(ctx context.Context, db pg.DB, sdb *sinkdb.DB) error {
 		case "network":
 			grant.Policy = "crosscore"
 		}
-		err = sdb.Exec(ctx, store.Save(ctx, &grant))
-		if err != nil {
-			return errors.Wrap(err)
-		}
+		stores = append(stores, store.Save(ctx, &grant))
```

Comment:
Looks like this does produce multiple writes to the same key if we have multiple access tokens with the same `Policy` (since our key is `/core/grant` + `Policy`), and right now that throws an error (if, for example, we try to migrate two different client access tokens)